Dockerfile

```diff
@@ -219,11 +219,18 @@ COPY . /work
 WORKDIR /work
 RUN ./pkg/rpm/build.sh
 
-FROM opensuse/leap:15.3 AS sles15-build
-
-RUN set -x; zypper -n install git systemd autoconf automake flex libtool libcurl-devel libopenssl-devel libyajl-devel gcc gcc-c++ zlib-devel rpm-build expect cmake systemd-devel systemd-rpm-macros java-11-openjdk-devel unzip zip && \
-    # Add home:d4vid:co22 repo to install >3.4 bison
-    zypper addrepo https://download.opensuse.org/repositories/home:/d4vid:/co22/15.3/home:d4vid:co22.repo && \
+FROM opensuse/leap:15.1 AS sles15-build
+SHELL ["/bin/bash", "-c"]
+RUN set -x; zypper -n install git systemd autoconf automake flex libtool libcurl-devel libopenssl-devel libyajl-devel gcc gcc-c++ zlib-devel rpm-build expect cmake systemd-devel systemd-rpm-macros java-11-openjdk-devel unzip zip
+# Add agent-vendor.repo to install >3.4 bison
+RUN echo $'[google-cloud-monitoring-sles15-x86_64-test] \n\
```

Comment:
Can we host this repo file somewhere on Google's side as well? Not a blocker, just curious.

Reply:
Rapture repo is google-owned and also public. Though it will soon be deprecated.

Reply:
Sorry I meant just the text that we're echoing to `agent-vendor.repo`, rather than the package repo itself. If we host `agent-vendor.repo` then we don't have to echo to the file locally.

Reply:
Creating any file under /etc/yum.repo.d will need extra permission and steps. I found it easy to do it this way.